.github/workflows/publish_image.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy to ghcr.io
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
should we trigger this on a tag push? we deploy on tag push

Reply:
agreed, added the change